source/operations.rst

```diff
@@ -0,0 +1,354 @@
+   +--------------------------------------+-------------------------+--------------------------------------+
+
+
+In this case we will use ``public00`` as the pool for public IPs and 
+``1aaf20b6-47a1-47ef-972e-7b36872f678f`` as the network id. Check with the provider
+which is the right network to use. Use these values in the ``terraform.tfvars``
+file:
+
+.. code-block:: terraform
+
+   ip_pool = "public00"
+   net_id  = "1aaf20b6-47a1-47ef-972e-7b36872f678f"
+
+
+You may want to check the right flavors for your VMs and adapt other variables
+in ``terraform.tfvars``. To get a list of flavors you can use:
+
+.. code-block:: shell
+
+   $ openstack flavor list
+   +--------------------------------------+----------------+-------+------+-----------+-------+-----------+
+   | ID                                   | Name           |   RAM | Disk | Ephemeral | VCPUs | Is Public |
+   +--------------------------------------+----------------+-------+------+-----------+-------+-----------+
+   | 26d14547-96f2-4751-a686-f89a9f7cd9cc | cor4mem8hd40   |  8192 |   40 |         0 |     4 | True      |
+   | 42eb9c81-e556-4b63-bc19-4c9fb735e344 | cor2mem2hd20   |  2048 |   20 |         0 |     2 | True      |
+   | 4787d9fc-3923-4fc9-b770-30966fc3baee | cor4mem4hd40   |  4096 |   40 |         0 |     4 | True      |
+   | 58586b06-7b9d-47af-b9d0-e16d49497d09 | cor24mem62hd60 | 63488 |   60 |         0 |    24 | True      |
+   | 635c739a-692f-4890-b8fd-d50963bff00e | cor1mem1hd10   |  1024 |   10 |         0 |     1 | True      |
+   | 6ba0080d-d71c-4aff-b6f9-b5a9484097f8 | small          |   512 |    2 |         0 |     1 | True      |
+   | 6e514065-9013-4ce1-908a-0dcc173125e4 | cor2mem4hd20   |  4096 |   20 |         0 |     2 | True      |
+   | 85f66ce6-0b66-4889-a0bf-df8dc23ee540 | cor1mem2hd10   |  2048 |   10 |         0 |     1 | True      |
+   | c4aa496b-4684-4a86-bd7f-3a67c04b1fa6 | cor24mem50hd50 | 51200 |   50 |         0 |    24 | True      |
+   | edac68c3-50ea-42c2-ae1d-76b8beb306b5 | test-bigHD     |  4096 |  237 |         0 |     2 | True      |
+   +--------------------------------------+----------------+-------+------+-----------+-------+-----------+
+
+Finally ensure your public ssh key is also listed in the ``cloud-init.yaml``
+file and then you are ready to deploy the cluster with:
+
+.. code-block:: shell
+
+   $ terraform apply
+
+Your VMs are up and running, it's time to get kubernetes configured and running
+with ansible:
+
+.. code-block:: shell
+  
+   $ cd ..   # you should be now in <new provider>
+   $ ANSIBLE_TRANSFORM_INVALID_GROUP_CHARS=silently TF_STATE=./terraform \
+     ansible-playbook --inventory-file=$(which terraform-inventory) \
+     playbooks/k8s.yaml
+
+
+Interacting with the cluster
+::::::::::::::::::::::::::::
+
+As the master will be on a private IP, you won't be able to directly interact
+with it, but you can still ssh into the VM using the ingress node as a bastion
```

Comment:
Not sure bastion will mean something to all maybe something like gateway or relay will be more clear?